routes/users.routes.js

```diff
@@ -58,4 +59,39 @@ router.post("/signup", validateEmpty, validPwd, (req, res) => {
   });
 });
 
+
+/* GET/ signin */
+router.get("/signin", (req, res, next) => {
+  res.render("users/signin.hbs");
+});
+
+/* POST/ singin */
+router.post("/signin", (req, res, next) => {
+  const { username, password } = req.body
+  User.findOne({ username })
+    .then((user) => {
+      if (!user) {
+        res.render("users/signin.hbs", { msg: 'Oh no, something went wrong! Please check again if your username or password are correct!' })
+      }
+      else {
+        bcrypt.compare(password, user.password)
+          .then((isMatching) => {
+            if (isMatching) {
+              //req.session.userInfo = user
+              //req.app.locals.isUserLoggedIn = true
+
+              res.redirect(`/profile/${user._id}`)
```

Comment:
Sorry I messed up the routes in the trello card. it should be: 

```suggestion
res.redirect(`/profiles/${user._id}`)
```

Reply:
haha no worries!